packages/twenty-front/src/modules/object-record/record-store/contexts/RecordFieldValueSelectorContext.tsx

```diff
@@ -1,78 +1,16 @@
-import { Dispatch, SetStateAction, useState } from 'react';
-import { createContext, useContextSelector } from 'use-context-selector';
-
-import { ObjectRecord } from '@/object-record/types/ObjectRecord';
-
-export type RecordFieldValue = {
-  [recordId: string]: {
-    [fieldName: string]: any;
-  };
-};
-
-export const RecordFieldValueSelectorContext = createContext<
-  [RecordFieldValue, Dispatch<SetStateAction<RecordFieldValue>>]
->([{}, () => {}]);
-
-export const useSetRecordValue = () => {
-  const setTableValue = useContextSelector(
-    RecordFieldValueSelectorContext,
-    (value) => value[1],
-  );
-
-  return (recordId: string, newRecord: any) => {
-    setTableValue((currentTable) => ({
-      ...currentTable,
-      [recordId]: newRecord,
-    }));
-  };
-};
-
-export const useRecordValue = (recordId: string) => {
-  const tableValue = useContextSelector(
-    RecordFieldValueSelectorContext,
-    (value) => value[0]?.[recordId],
-  );
-
-  return tableValue as ObjectRecord | undefined;
-};
+import { recordStoreFamilySelector } from '@/object-record/record-store/states/selectors/recordStoreFamilySelector';
+import { useRecoilValue } from 'recoil';
 
 export const useRecordFieldValue = <T,>(
   recordId: string,
   fieldName: string,
 ) => {
-  const recordFieldValue = useContextSelector(
-    RecordFieldValueSelectorContext,
-    (value) => value[0]?.[recordId]?.[fieldName],
+  const recordFieldValue = useRecoilValue(
```

Comment:
why don't we completely delete this component?

Reply:
I'm going to finish this refactor after the current sprint.